[2023-09-21T00:00:01.142+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: create_ML_customerchurn.create_user_table scheduled__2023-09-20T00:00:00+00:00 [queued]>
[2023-09-21T00:00:01.155+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: create_ML_customerchurn.create_user_table scheduled__2023-09-20T00:00:00+00:00 [queued]>
[2023-09-21T00:00:01.155+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-21T00:00:01.156+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-21T00:00:01.157+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-21T00:00:01.172+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_user_table> on 2023-09-20 00:00:00+00:00
[2023-09-21T00:00:01.183+0000] {standard_task_runner.py:55} INFO - Started process 22179 to run task
[2023-09-21T00:00:01.187+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'create_ML_customerchurn', 'create_user_table', 'scheduled__2023-09-20T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/copy_db.py', '--cfg-path', '/tmp/tmpimmzt8si']
[2023-09-21T00:00:01.188+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask create_user_table
[2023-09-21T00:00:01.259+0000] {task_command.py:376} INFO - Running <TaskInstance: create_ML_customerchurn.create_user_table scheduled__2023-09-20T00:00:00+00:00 [running]> on host 411713a30e55
[2023-09-21T00:00:01.332+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_ML_customerchurn
AIRFLOW_CTX_TASK_ID=create_user_table
AIRFLOW_CTX_EXECUTION_DATE=2023-09-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-20T00:00:00+00:00
[2023-09-21T00:00:01.343+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-09-21T00:00:01.348+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
                first_name TEXT NULL,
                last_name TEXT NULL,
                email TEXT NULL,
                street_number SMALLINT NULL,
                street_name TEXT NULL,
                surburb TEXT NULL,
                post_code INTEGER NULL,
                state TEXT NULL, 
                country TEXT NULL
            );
        , parameters: None
[2023-09-21T00:00:01.350+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-09-21T00:00:01.362+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=create_ML_customerchurn, task_id=create_user_table, execution_date=20230920T000000, start_date=20230921T000001, end_date=20230921T000001
[2023-09-21T00:00:01.399+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-21T00:00:01.469+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
